.github/scripts/git_issues/generate_jira_issue_data.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Github issues Epic key
+jira_fields='{"customfield_10014": "ASM-7343"}'
+
+repository=$(echo "$GITHUB_CONTEXT" | jq '.repository' --raw-output)
+git_user=$(echo "$GITHUB_CONTEXT" | jq '.actor' --raw-output)
+git_issue_id=$(echo "$GITHUB_CONTEXT" | jq '.event.issue.id' --raw-output)
+git_issue_title=$(echo "$GITHUB_CONTEXT" | jq '.event.issue.title' --raw-output)
+git_issue_body=$(echo "$GITHUB_CONTEXT" | jq '.event.issue.body' --raw-output)
+git_issue_url=$(echo "$GITHUB_CONTEXT" | jq '.event.issue.html_url' --raw-output)
+git_issue_created_at=$(echo "$GITHUB_CONTEXT" | jq '.event.issue.created_at' --raw-output)
+git_issue_updated_at=$(echo "$GITHUB_CONTEXT" | jq '.event.issue.updated_at' --raw-output)
+git_issue_comment_url=$(echo "$GITHUB_CONTEXT" | jq '.event.comment.html_url' --raw-output)
+git_issue_comment_body=$(echo "$GITHUB_CONTEXT" | jq '.event.comment.body' --raw-output)
+git_issue_comment_created_at=$(echo "$GITHUB_CONTEXT" | jq '.event.comment.created_at' --raw-output)
+git_issue_labels=$(echo "$GITHUB_CONTEXT" | jq '.event.issue.labels[].name' --raw-output)
+
+while IFS='' read -r line; do git_issue_labels_list+=("$line"); done < <(echo "${git_issue_labels}")
+if [[ "${git_issue_labels_list[*]}" == *"ASM"* ]]; then
+  akeyless_jira_issue=$(echo "${git_issue_labels_list[*]}" | grep -e "ASM" )
+  echo "Jira issue exist: ${akeyless_jira_issue}"
+  jira_issue_exist="true"
+else
+  echo "No Jira issue label found"
+  jira_issue_exist="false"
+fi
+
+echo "Generating Jira issue summary"
+jira_issue_summary="Github Issue: ${git_issue_title}"
+echo "Jira issue summary: ${jira_issue_summary}"
+
+if [[ "${git_issue_event_name}" == "issues" ]]; then
+  if [[ ${git_issue_event_action} == "opened" ]]; then
+    echo "Generating new jira issue description"
+    JIRA_EVENT_DESCRIPTION="*Issue Title:* ${git_issue_title}
+    *Repository:* ${repository}
+    *Git Issue id:* ${git_issue_id}
+    *Git User:* ${git_user}
+    *Git issue Description:*
+    ${git_issue_body}
+    *Git Issue Creation Time:* ${git_issue_created_at}
+    *Git Issue URL:* ${git_issue_url}"
+
+  elif [[ ${git_issue_event_action} == "closed" ]]; then
+    echo "Closing ${akeyless_jira_issue} jira issue description"
+    JIRA_EVENT_DESCRIPTION="*Git Issue has been closed*
+    *Git issue closed by:* ${git_user}
+    *Git Issue Update Time:* ${git_issue_updated_at}
+    *Git Issue URL:* ${git_issue_url}"
+  fi
+elif [[ ${git_issue_event_name} == "issue_comment" ]]; then
+  echo "Generation new comment on jira issue ${akeyless_jira_issue}"
+  JIRA_EVENT_DESCRIPTION="*Git Issue has new comment*
+  *Git User:* ${git_user}
+  *Git issue comment:*
+  ${git_issue_comment_body}
+  *Git Issue Comment Creation Time:* ${git_issue_comment_created_at}
+  *Git Issue Comment URL:* ${git_issue_comment_url}"
+fi
+
+delimiter="$(openssl rand -hex 8)"
```

Comment:
Why are you using this delimiter?

Reply:
method that i worked with on slack message the generate random delimiter the context for markdown

Reply:
But why it's needed?

Reply:
this is preparation for slack message if needed